katalog/thanos/thanos/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../thanos-components/thanos-query
+# - ../thanos-rule
```

Comment:
Avoid commented code. It can confuse users

Reply:
as explained above, i decided to leave this commented because if we need in the future to configure the thanos-rule, is alredy there, but yes, can definetely be removed.